child-process-worker.js

```diff
@@ -1,13 +1,17 @@
 const path = require('path')
 const childProcess = require('child_process')
 
-const WORKER_PATH = path.join(__dirname, 'worker.js')
+var WORKER_PATH = '/tmp/worker.js' //path.join(__dirname, 'worker.js')
+
+const WorkerMain = require('./worker')
+require('fs').writeFileSync('/tmp/worker.js', ';('+WorkerMain.toString()+')();function __name (){}; ')
```

Comment:
note the `__name` noop is needed because that function is inserted by the esbuild.